app/views/resources/iiif.json.jbuilder

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+json.set! :@context, 'http://iiif.io/api/presentation/2/context.json'
```

Comment:
I'm good with this as it is, but do we want to try and v3 instead?

Reply:
🤷‍♂️ maybe? i was just copying from exhibits

Reply:
Ah.. maybe not a good idea since it wouldn't yet be consumable by Spotlight?